### PR TITLE
Unreleased version of khmer

### PR DESCRIPTION
**Pull request by standage**
There's already a recipe for khmer, but I'd like to add the latest Github master until a new release is published

* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [x] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

**Comment by standage** [comment]
cc @luizirber @camillescott

File: recipes/khmer/fe0ce11645/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
+{% set version = "fe0ce116456b296c522ba24294a0cabce3b2648b" %}
+{% set sha256 = "b6da62bb477414a59fa1283e6c00a4ecf96728b1ce7981ec2aaa1058987974d0" %}
+
+package:
+  name: khmer
+  version: '{{version}}'
```
Comment (bgruening) [inline recipes/khmer/fe0ce11645/meta.yaml]: I don't think this is a good idea. Its not clear which version conda will favor and is the "latest". Please ask for a release instead ...

Comment (epruesse) [reply recipes/khmer/fe0ce11645/meta.yaml]: Call it 3.0.0a2 if you must ... but that won't change that everyone will get it on their desk.

The thing is that Bioconda has no mechanism for distributing previews at this time.

**Comment by epruesse** [comment]
Closing in favor of #12360

**Pull request closed without merging**